tests/nix_tests/configs/osmosis-outpost.nix

```diff
@@ -0,0 +1,12 @@
+{ pkgs ? import ../../../nix { } }:
+let evmosd = (pkgs.callPackage ../../../. { });
+in
+evmosd.overrideAttrs (oldAttrs: {
+  # Patch the evmos binary to:
+  # - use the CrossChainSwap contract address in the testing setup
+  # - update the corresponding IBC channel to match the tests setup
+  patches = oldAttrs.patches or [ ] ++ [
+    ./osmosis-outpost.patch
+  ];
+})
+
```

Comment:
should this stuff be included? 👀

Reply:
yes we still need it if we want to keep testing with XCS v2 directly building an IBC transfer

Reply:
no, this was removed

Reply:
This patch is no longer necessary. You can delete this